### DISCUSSION TITLE
Frequency shifting

### DISCUSSION THREAD

**Discussion by barisdinc** [kaefe64/Arduino_uSDX_Pico_FFT_Proj#5] [General]
Hi @kaefe64 Klaus,
I am OH2UDS/TA7W and I am working on a VHF/UHF/SHF SDR project (mainly for a portable QO-100 station). A brief info is in https://www.youtube.com/watch?v=xVOr04rAqmU 

I prepared the prototypes and using a software based on your Arduino uSDX Pico but using Analog Devices IQ demodulators and modulators as hardware.

On hardware side in the prototypes I used a wrong PLL/VCO which only can step 10 KHz, so I am looking for the solution to shift the listening channel using SDR capabilities.

In Arjante's code the shifting and filtering part is easy and I can select the FFT (frequency) BINs that I would like to listen. But in your implementation I was not able to figure out which part is doing the shifting of the desired channel to 3.9Khz Fc and demodulationg that signal.

I will be very happy if you can support me on this,

BTW: Our project is also fully opensource and details can be found on these repositories. I would like to ask your permission to use your firmware for the starting point of our project which gave us a lot of inspirations to start.

[https://github.com/barisdinc/N3B_SDR_Transceiver_FW](https://github.com/barisdinc/N3B_SDR_Transceiver_FW)
[https://github.com/barisdinc/N3B_SDR_Transceiver_HW](https://github.com/barisdinc/N3B_SDR_Transceiver_HW)

73's de 
Baris DINC
OH2UDS

**Comment by kaefe64** [comment]
Hi Baris

I answered at "Issues" section.

73
Klaus F
PY2KLA

**Comment by barisdinc** [comment]
Thank you so much
